supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/api/PipelinesApiRequestManager.scala

```diff
@@ -96,7 +96,13 @@ class PipelinesApiRequestManager(val qps: Int Refined Positive, requestWorkers:
   }
 
   def monitorQueueSize() = {
-    val load = if (workQueue.size > LoadConfig.PAPIThreshold) HighLoad else NormalLoad
+    val load = if (workQueue.size > LoadConfig.PAPIThreshold) {
+      log.warning(s"PAPI Request Manager notifying HighLoad with queue size ${workQueue.size} exceeding limit of ${LoadConfig.PAPIThreshold}")
```

Comment:
I'm curious why this is WARN and the new IoActor message is INFO - is this one scarier?

Reply:
I do think PAPI backups are much less common and therefore more suspicious. That said, I am making the `IoActor` one a warning too for consistency.